src/commands/buildCommands/configs/vendorBundlesConfig.js

```diff
@@ -0,0 +1,98 @@
+const ManifestPlugin = require('webpack-manifest-plugin');
```

Comment:
Copied from [`mwp-config`](https://github.com/meetup/meetup-web-platform/blob/master/packages/mwp-config/webpack/vendorBundlesConfig.js), and edited for Webpack 4